src/Components/Autocomplete.fs

```diff
@@ -129,11 +143,33 @@ let createHelptextToolTip (overloads:DTO.OverloadSignature[]) (position:JQueryCo
 
         let! evt = Async.AwaitObservable helptextEvent.Publish
         match evt with
-        | Next by -> return! loop ((i + by + overloads.Length) % overloads.Length)
+        | Show text ->
+            return! async {
```

Comment:
You shouldn't need `return! async { <stuff> }`. It should be exactly the same as writing just `<stuff>`.